### PR TITLE
Adjust sample data

### PR DESCRIPTION
**Pull request by nilsthamm**
refs #500

### CODE REVIEW

File: db/sample_data/events.rb [hunk 1]
```diff
@@ -124,3 +124,24 @@ def event_bachlorpodium
...
+      max_participants: 25,
+      organizer: 'HPI Schülerklub',
+      knowledge_level: 'Fortgeschrittene',
+      date_ranges: [date_range_summer_camp],
+      application_deadline: Date.tomorrow,
+      application_status_locked: false,
```
Comment (cmfcmf) [inline db/sample_data/events.rb]: This field has recently been split into two and is the reason why the tests fail.

Comment (nilsthamm) [reply db/sample_data/events.rb]: thank you, i will fix it

**Comment by tobiasduerschmid** [comment]
@nilsthamm: The build on this branch is still broken.

File: db/sample_data.rb [hunk 2]
```diff
@@ -67,8 +68,7 @@ def add_sample_data
   [events, users, profiles, application_letters, requests, agreement_letters].each do |models|
     save_models(models)
   end
-
-  # set deadline to past to work around validation of application letters
```
Comment (bjrne) [inline db/sample_data.rb]: Why did you remove this comment? I think it's important to understand why we already save things here.

**Review by bjrne — APPROVED** [review]
Looks good now. I added two very small commits, please have a look at them before merging. Otherwise this looks good 🔪

**Comment by nilsthamm** [comment]
@bjrne thanks for the review, i think i deleted the comment by accident...
